PowerShellGet/PSModule.psm1

```diff
@@ -8582,12 +8595,21 @@ function ValidateAndAdd-PSScriptInfoEntry
         # Validate the property value and also use proper key name as users can specify the property name in any case.
         $script:Version {
                             $KeyName = $script:Version
-
                             [Version]$Version = $null
-
-                            if([System.Version]::TryParse($Value, ([ref]$Version)))
+                            
+                            # Special case for prerelease string in version field, must pull out before parsing version
+                            if ( ( $Value -match '-' ) -and 
+                                 ( [System.Version]::TryParse($($Value -split '-',2 | Select-Object -First 1), ([ref]$Version)) ) -and 
+                                 ( Validate-PrereleaseString -Version $($Value -split '-',2 | Select-Object -First 1) `
+                                                                                     -Prerelease $($Value -split '-',2 | Select-Object -Skip 1) `

```

Comment:
I thought I had fixed this, I must have missed it.  I will fix it now, thanks for finding it.

---
In reply to: [132584316](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132584316) [](ancestors = 132584316,126589514)